### PR TITLE
Feature news overlay

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -224,18 +224,39 @@ Site.Gallery = {
...
 
 Site.News = {
-  $news: undefined,
-
   init: function() {
     var _this = this;
 
     _this.$news = $('#news');
+    _this.$overlay = $('#news-overlay');
+    _this.$overlayContent = $('#news-overlay-content');
```
Comment (mixcas) [inline js/main.js:233]: good practice :)

File: lib/functions-filters.php [hunk 2]
```diff
@@ -10,4 +10,15 @@ function add_slug_body_class( $classes ) {
   }
   return $classes;
 }
-add_filter( 'body_class', 'add_slug_body_class' );
\ No newline at end of file
+add_filter( 'body_class', 'add_slug_body_class' );
+
+// Remove <p> tags around images and embeds in wp_content
```
Comment (mixcas) [inline lib/functions-filters.php:15]: why all of these code?

Comment (varffs) [inline lib/functions-filters.php:15]: check out the styling. need to remove `<p>` so you can have images with margins outside the paragraphs like features in 8106

Comment (mixcas) [inline lib/functions-filters.php:15]: i c

**Comment by mixcas** [comment]
@varffs check my code and  merge

**Comment by varffs** [comment]
you're right tho needs images ready. I can do that

**Comment by varffs** [comment]
@mixcas imagesLoaded and also scroll fix. plz check thru and merge

**Pull request merged**